go.mod

```diff
@@ -3,15 +3,15 @@ module github.com/splunk/splunk-operator
 go 1.13
 
 require (
-	github.com/aws/aws-sdk-go v1.17.7
+	github.com/aws/aws-sdk-go v1.30.12
```

Comment:
Do we need this line as this is already being covered as part of https://github.com/splunk/splunk-operator/pull/267/files ?

Reply:
This version is updated automatically by requiring prometheus-operator v0.38.3 so it is necessary for this standalone fix. When merging this and CSPL-841 we will need to ensure that the latest version of the awk-sdk-go library is retained (depending on the order the tickets are merged in).